rocAL/rocAL/source/parameters/parameter_crop.cpp

```diff
@@ -57,8 +57,6 @@ void CropParam::array_init()
     croph_arr_val.resize(batch_size);
     x2_arr_val.resize(batch_size);
     y2_arr_val.resize(batch_size);
```

Comment:
why is in_width removed? Is there anything else which holds this information

Reply:
The in_width and in_height is replaced by the in_roi that is set